src/cryptography/hazmat/bindings/openssl/_conditional.py

```diff
@@ -268,6 +268,11 @@ def cryptography_has_get_proto_version():
         "SSL_CTX_get_max_proto_version",
         "SSL_get_min_proto_version",
         "SSL_get_max_proto_version",
+
```

Comment:
this should add a closing square bracket here:   ']'

Reply:
Nice catch.